### PR TITLE
add user account

### PR DESCRIPTION
**Pull request by B-3PO**
add new user account endpoints

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,89 @@ response:
...
+
+let body = {
+  name_on_account: 'Bob TheBuilder',
+  account_number: '123',
+  routing_number: '1234',
+  currency: 'USD',
+  bank_country: 'US'
+};
+
+rf.createUserAccounts(body)
```
Comment (ben-routefusion) [inline README.md:172]: Can we rename this `createUserAccount`? All of the other APIs in this SDK use a singular resource naming convention when dealing with a single resource, e.g. create, get (one record)

Comment (ben-routefusion) [reply README.md:172]: Looks like all three functions return only one object, so they should all be singular names.

Comment (B-3PO) [reply README.md:172]: i kept the names in line with the endpoints but i can change them. The one endpoint returns an array of 1 item. The reason being to confirm to the future of multiple accounts. I will update the singular endpoint

Comment (B-3PO) [reply README.md:172]: updated

Comment (ben-routefusion) [reply README.md:172]: Ok, just need to update the README to match now, it still has the plurals in it.

Comment (B-3PO) [reply README.md:172]: thanks

File: README.md [hunk 2]
```diff
@@ -156,6 +156,89 @@ response:
...
+```js
+const rf = require('routefusion-sdk').Instance();
+
+// This only returns one account
+rf.getUserAccounts(userUuid)
+  .then(resp => resp)
+  .catch(err => err)
+```
+response:
+```json
```
Comment (ben-routefusion) [inline README.md:194]: Is this supposed to be an array? You said it's an array of one, right?

File: README.md (hunk 1 above)
Comment (B-3PO) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (B-3PO) [reply README.md:194]: it is now :)

**Pull request merged**